apps/manifest.go

```diff
@@ -12,6 +12,12 @@ import (
 // Where static assets are.
 const StaticFolder = "static"
 
+// Root Call path for incoming webhooks from remote (3rd party) systems. Each
+// webhook URL should be in the form:
+// "{PluginURL}/apps/{AppID}/webhook/{PATH}...?secret=XYZ", and it will invoke a
+// Call with "/webhook/{PATH}"."
+const PathWebhook = "/webhook"
```

Comment:
Should we make this configuration like `Bindings`? Maybe we need a followup ticket.

Reply:
@hanzei yes we should file a ticket, but I intentionally did not since I think we should figure out a greater scheme of attaching "Expand" and "State" to calls.